### PR TITLE
Feat/services action menu

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-4629
| License          | BSD 3-Clause

## Description

* Create component to handle billing actions
* Use component in the hub and dedicated 

## Related 
- [X] 2API /hub
- [x] TRANS-26562

### CODE REVIEW

File: packages/manager/modules/billing/src/components/services-actions/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import '@ovh-ux/ng-translate-async-loader';
```
Comment (JeremyDec) [inline packages/manager/modules/billing/src/components/services-actions/index.js:2]: Don't you require this dep in your peer ones ?

File: packages/manager/modules/billing/src/components/services-actions/services-actions.controller.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  $onInit() {
+    this.warningLink = `${this.autorenewLink}/warn-nic?nic=${this.service.contactBilling}`;
+    this.billingLink = this.RedirectionService.getURL('billing');
+    this.updateLink = `${this.autorenewLink}/update?serviceId=${this.service.serviceId}`;
+    this.cancelResiliationLink = `${this.autorenewLink}/cancel-resiliation?serviceId=${this.service.serviceId}`;
+
+    switch (this.service.serviceType) {
+      case 'EXCHANGE':
```
Comment (JeremyDec) [inline packages/manager/modules/billing/src/components/services-actions/services-actions.controller.js:20]: Why not having all these service types in your constants file?

Comment (marie-j) [reply packages/manager/modules/billing/src/components/services-actions/services-actions.controller.js:20]: `serviceType` has only few chances to change but sure I'll put them in the constants file

Comment (JeremyDec) [reply packages/manager/modules/billing/src/components/services-actions/services-actions.controller.js:20]: it is just that I see that thoses names are used elsewhere, e.g.: https://github.com/ovh/manager/pull/2907/files#diff-ebe3111f85479ef61dfc6286d6c33db3R62 here.
But it is nitpicking

File: packages/manager/modules/billing/src/components/services-actions/index.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/billing/src/components/services-actions/index.js:2]: I forgot to add it, nice catch !

**Comment by marie-j** [comment]
Updated to add a check for the US where feature is not available

**Comment by cbourgois** [comment]
❤️ 
> * Use component in the hub and dedicated

**Comment by JeremyDec** [comment]
@marie-j conflict !

**Comment by marie-j** [comment]
Rebased to fix conflicts

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#13263.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#13263.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report build#13263.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#13263.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by marie-j** [comment]
Had to rebase to fix conflicts and perform update due to UIKit v4 migration

**Pull request merged**